src/main/certificateStore.js

```diff
@@ -26,27 +25,25 @@ function areEqual(certificate0, certificate1) {
 }
 
 function getHost(targetURL) {
-  return url.parse(targetURL).host;
+  const parsedURL = new URL(targetURL);
+  return parsedURL.origin;
 }
 
 function CertificateStore(storeFile) {
   this.storeFile = storeFile;
   let storeStr;
+  let jsonStore;
   try {
     storeStr = fs.readFileSync(storeFile, 'utf-8');
-    storeStr = Validator.validateCertificateStore(storeStr);
-    if (!storeStr) {
+    jsonStore = JSON.parse(storeStr);
```

Comment:
Do we need to do this twice? Seems like overkill since `validateCertificateStore` is already making sure we have an object.

Reply:
fixed, it is also cleaner, thanks!